cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -125,22 +130,41 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 			parent.Close()
 			return nil, err
 		}
+	} else if oci.IsJobContainer(s) {
+		// If we're making a job container fake a task (i.e reuse the wcowPodSandbox logic)
+		p.sandboxTask = newWcowPodSandboxTask(ctx, events, req.ID, req.Bundle, parent, "")
+		if err := events.publishEvent(
+			ctx,
+			runtime.TaskCreateEventTopic,
+			&eventstypes.TaskCreate{
+				ContainerID: req.ID,
+				Bundle:      req.Bundle,
+				Rootfs:      req.Rootfs,
+				IO: &eventstypes.TaskIO{
+					Stdin:    req.Stdin,
+					Stdout:   req.Stdout,
+					Stderr:   req.Stderr,
+					Terminal: req.Terminal,
+				},
+				Checkpoint: "",
+				Pid:        0,
+			}); err != nil {
+			return nil, err
```

Comment:
do we want to cleanup the sandbox task if the publish event call fails?

Reply:
Doesn't seem like we do anything for wcow if it fails. What can we do to cleanup even? There's nothing running at this point

Reply:
In that case then I don't see any issue